cypress/e2e/basic-starter.cy.ts

```diff
@@ -1,5 +1,30 @@
+beforeEach(() => {
+  cy.visit('/');
+});
+
 describe('Basic Starter', () => {
   it('has a navigation menu', () => {
-    cy.visit('/').get('header nav').should('exist.and.be.visible');
+    cy.get('header nav').should('exist.and.be.visible');
+  });
+  it('can navigate through the menu links', () => {
+    cy.get('header > div > nav > ul > li:nth-child(2) > a').click();
```

Comment:
I think this assertion is too coupled with the markup structure. I think it would be sufficient for us to check the menu link texts and that they can be used for navigating.